src/utils/LocalStorageData.ts

```diff
@@ -1,18 +1,15 @@
 
-export const getVasUsername = (): any =>{
+export const getVasUsernameLS = (): any =>{
     const username = localStorage.getItem('vasUsername')
-    if (username){
-        return {username:username, isLoggedIn: true}
-    }
-    return {username:"", isLoggedIn: false}
+    return username
 }
 
-export const setVasUsername = (username:string): boolean =>{
+export const setVasUsernameLS = (username:string): boolean =>{
```

Comment:
do use the proper understandable word next time, people might not understand what LS stands for. so instead use 
setVasUsernameLocalStorage 

make code as readable as possible